bin/bluey.cpp

```diff
@@ -1,6 +1,14 @@
 #include <iostream>
-
+#include "../src/Item.hpp"
```

Comment:
Move your .hpp files to **<Project_root>/include/sudoManager**.
Use `target_include_directories()`.

This line should look like this: `#include "sudoManager/Item.hpp"`